src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -123,11 +123,17 @@ public static void propagateIgnoreGlobList(Author author, List<String> ignoreGlo
         author.appendIgnoreGlobList(ignoreGlobList);
     }
 
+    /**
+     * Adds author to the {@code AuthorList}.
+     */
     public void addAuthor(Author author) {
         authorList.add(author);
         setAuthorDetails(author);
     }
 
+    /**
+     * Overloads {@code addAuthor} and {@code propagateIgnoreGlobList} methods.
```

Comment:
Overload method should repeat the original method's header and include the new information.
The phrase `Overloads {@code addAuthor}` doesn't help the user when she reads this javadoc alone.